Server/controlers/Entry.js

```diff
@@ -1,49 +1,60 @@
-import EntryModel from '../models/Entry';
-import entriesu from '../models/Entry';
-import uuidv4 from 'uuid/v4';
-import moment from 'moment';
-import validation from '../helpers/validation';
-import joi from 'joi';
-
+import EntryModel from "../models/Entry";
+import uuidv4 from "uuid/v4";
+import moment from "moment";
 class Entry {
-	static async NewEntry(req, res) {
-		joi.validate(req.body, validation.validator.Entrychema).then((result) => {		
-		const entry = {
-			id: uuidv4(),
-			title:req.body.title,
-			description: req.body.description,
-			email: req.user.email,
-			createdOn: moment(new Date())
-		};
-		const response = EntryModel.addNewEntry(entry);
-		res.status(200).json({
-			status: 200,
-			data: {
-			message: 'entry successfully created',
-	        response 
-			}
-		});
-	}).catch(error => res.send({
-		"status": 400,
-		"error" : {"message": error.details[0].message.replace(/[&\/\\#,+()$~%.'":*?<>{}]/g, '')}
-	}));
-}
-	static async GetEntry(req, res){
+  static async NewEntry(req, res) {
+    const entry = {
+      id: uuidv4(),
+      title: req.body.title,
+      description: req.body.description,
+      email: req.user[0].email,
+      createdOn: moment(new Date())
+    };
+    const response = await EntryModel.addEntry(entry);
+    return res.status(200).send({
+      status: 200,
+      data: {
+        message: "entry successfully created",
+        entry: response
+      }
```

Comment:
@ByiringiroYves You are not handling errors.

Reply:
but  Caleb you should check  server/middleware/users.js or  server/middleware/entries.js
it is where am catching errors you should check again